source/containerimage/ocilayout.go

```diff
@@ -0,0 +1,121 @@
+package containerimage
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"time"
+
+	"github.com/containerd/containerd/content"
+	"github.com/containerd/containerd/content/local"
+	"github.com/containerd/containerd/images"
+	"github.com/containerd/containerd/remotes"
+	"github.com/moby/buildkit/session"
+	sessioncontent "github.com/moby/buildkit/session/content"
+	"github.com/moby/buildkit/source"
+	ocispecs "github.com/opencontainers/image-spec/specs-go/v1"
+)
+
+const (
+	ociImageIndexFile = "index.json"
+)
+
+// GetOCILayoutResolver gets a resolver to an OCI layout for a specified scope from the pool
+func GetOCILayoutResolver(layoutPath string, sm *session.Manager, sessionID string, g session.Group) *OCILayoutResolver {
+	fileStore, err := local.NewStore(layoutPath)
```

Comment:
Is this some old code? No local contentStores on the daemon side.

Reply:
Yeah, that was just a placeholder until I knew where to move it client-side. Going to remove right now.

Reply:
Done